projects/plugins/jetpack/modules/videopress/class.videopress-edit-attachment.php

```diff
@@ -192,6 +193,16 @@ public function fields_to_edit( $fields, $post ) {
 			'show_in_edit'  => false,
 		);
 
+		if ( $file_statuses['hd_1080p_compat'] ) {
+			$fields['videopress_original_file_url'] = array(
+				'label'         => _x( 'Original File URL', 'A header for the original file url field', 'jetpack' ),
+				'input'         => 'html',
+				'html'          => "<input type=\"text\" name=\"videopress_original_file_url\" value=\"{$original_file_url}\" readonly=\"readonly\"/>",
```

Comment:
Should $original_file_url be escaped ?

Reply:
The url gets sanitized when we process the file, but probably doesn't hurt to escape it here just to be completely certain.